jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/test/java/org/jbpm/console/ng/pr/client/test/list/MyDataSetProcessInstanceWithVariablesListPresenter.java

```diff
@@ -89,7 +88,7 @@ public boolean onError(ClientRuntimeError error) {
     }
 
     @Override
-    protected DataSetReadyCallback createDataSetDomainSpecificCallback(int startRange, int totalRowSize, List<ProcessInstanceSummary> instances) {
+    protected DataSetReadyCallback createDataSetDomainSpecificCallback(int startRange, int totalRowSize, List<ProcessInstanceSummary> instances, FilterSettings tableSettings) {
```

Comment:
That is because a change in the method signature from the parent class. This mock class actually ignores all three parameters, apparently not relevant for the test condition.